pynxtools_xps/vms/vamas.py

```diff
@@ -46,14 +52,26 @@ class VamasMapper(XPSMapper):
     config_file = "config_vms.json"
 
     def __init__(self):
-        self.file = None
-        self.parsers = [
-            VamasParserRegular,
-            VamasParserIrregular,
-        ]
-        self.parser_map = {
-            "regular": VamasParserRegular,
-            "irregular": VamasParserIrregular,
+        self.parsers: List[Any] = []
+
+        self.units: dict = {
```

Comment:
This could also be a top-level constant, no? Or do you add something while parsing?

Reply:
I moved it out, thanks for the idea.